packages/plugins/users-permissions/admin/src/pages/Roles/pages/ListPage/index.js

```diff
@@ -91,7 +90,7 @@ const RoleListPage = () => {
 
   const handleNewRoleClick = () => {
     trackUsage('willCreateRole');
-    push(`/settings/${pluginId}/roles/new`);
+    push(`/settings/users-permissions/roles/new`);
   };
```

Comment:
if we're tidying this up, i think we should change whatever element is a button to be an anchor? the track usage can still be fired iirc.